Added initial content for  finaid page (#1028)
EuroPython/website: src/content/pages/finaid.mdx

---
title: Financial Aid
subtitle:
  As part of our commitment to the Python community, we offer special grants for
  people in need of financial aid to attend EuroPython. You’ll find all the
  information you need to apply for such a grant in this page.
---

# Financial Aid

As part of our commitment to the Python community, we are pleased to announce
that we offer special grants for people in need of financial aid to attend
EuroPython.

You’ll find all the information you need to apply for such a grant below.

---

## Financial Aid Sponsors

<div className="finaid-sponsor">
  <a href="https://www.europython-society.org/" target="_blank">
    <img src="/img/eps-logo.png" alt="Logo of the EuroPython Society" />
  </a>
</div>

> The EuroPython Society is sponsoring the Financial Aid Programme with €25,000 this year.

Reviewer: QQ: Is the 25K is correct for this year too? just in case is copied from last year

Reviewer 2: Thanks for noticing! It's already more, and we're working on increasing it :) 

I think we can also just skip it from the description for now, because it's mostly a not-actionable implementation detail at this point. We can include the actual amount given out in the post-conference report.

Author: checked with the previous conversations and amount is 30K as per our internal discussion to start with, I've updated the same and anyway like @artcz mentioned we can update the actual amount in the post-conference reports.

Reviewer 3: removed the grants figure in https://github.com/EuroPython/website/pull/1037/commits/8225457aaf0b14726ec7fbb45b25eb7f36d5170f as @artcz suggested.

If you want to support EuroPython and its efforts to make EuroPython accessible
to everyone, please consider sponsoring EuroPython by reaching out to

> [sponsoring@europython.eu][email-sponsoring].

Reviewer 3: ```suggestion
[sponsors@europython.eu][email-sponsoring].
```
We are advertising the helpdesk email for sponsors this year which is slightly different :)

Author: thanks for the thoughts, I've committed these changes.

1. **Free Ticket Voucher Grant:** Get a voucher for a free standard ticket for
   the in-person conference.
2. **Travel/Accommodation Grant:** We will reimburse travel and/or accommodation
   costs up to €400.
3. **Visa Application Fee Grant:** Get reimbursed for the costs of a short-stay
   Schengen visa to the Czech Republic (up to €80).

## Grant Eligibility

Our grants are open to everyone in need of financial support to participate in
EuroPython. We will look at the following criteria to help decide who qualifies
for a grant:

1. **Contributors:** We want to support those who make the Python community
   a positive, inclusive and welcoming space. We welcome grant applications
   from folks who need financial support and who clearly put effort into
   the community and EuroPython, including speakers, volunteers, community
   organizers, and open-source contributors.
2. **Social & Economic factors:** Everyone should have a chance to attend
   EuroPython, regardless of their social or economic situation.
3. **Diversity:** We want EuroPython to be the most diverse and inclusive event
   possible, and we especially want to help those from under-represented groups

>    in tech feel welcomed, be present in and contribute to our community.

Reviewer 3: ```suggestion
3. **Diversity:** We want EuroPython to be the most diverse and inclusive event possible, and we are especially committed to helping those from underrepresented groups in tech feel welcome, participate fully, and contribute to our community.
```
"Be present in" is quite awkward. Small rewording suggestion.

Author: thanks for the thoughts, I've committed these changes.

        alt="round two" />
    </td>
    <td>Deadline for submitting second-round applications[^round-1-vs-round-2]</td>
  </tr>
  <tr>
    <td>4 June 2025</td>
    <td className="finaid-accent">
      <img
        src="/img/accents/finaid-round2.svg"
        className="findaid-accent-round-marker"
        alt="round two" />
    </td>
    <td>Second round of grant notifications</td>
  </tr>
  <tr>
    <td>11 June 2025</td>
    <td>
      <img
        src="/img/accents/finaid-round2.svg"
        className="findaid-accent-round-marker"
        alt="round two" />
    </td>
    <td>Deadline to accept a second-round grant</td>
  </tr>
  <tr>
    <td>20 July 2025</td>
    <td></td>
    <td>Deadline for submitting receipts/invoices</td>
  </tr>
</tbody>
</table>

[^round-1-vs-round-2]: If you've already applied for round 1, you don't have to
reapply for round 2. Your application will automatically be considered for round
2 if you did not receive a grant in round 1.

---

## Grant Policies

### Policies for all grants

- Please note that all grants and grant offers are personal and
  non-transferable.

- You must accept a grant offer before the relevant acceptance deadline stated

>   in the timeline. The grant offer will expire otherwise.

Reviewer: nit: We could link the `timeline` section here

Author: okay, I'll try to add the timeline here

- The Financial Aid Team may also make grant offers outside the two main rounds
  of selection.

### Free Ticket Voucher Grant

- You will receive an **individual voucher code** for a free **regular**

>   conference ticket that gives you access to the three main conference days.

Reviewer: Gives you access to the sprint days too right?

Author: I'll check with the team and confirm back

Reviewer 3: Clarified sprints are included in https://github.com/EuroPython/website/pull/1037/commits/c2302bebd0ce0dbccbfc24deca1f45246693fe97

### Travel/Accommodation Grant & Visa Application Fee Grant

If you received a travel/accommodation grant or a visa application fee grant,
you will be reimbursed **after the conference** via PayPal or bank transfer
(SEPA/Swift). We will only reimburse you after we've received your
receipt/invoices and a **photo of your conference badge**.

In addition, please pay close attention to the following:

- The total combined travel/accommodation grant cannot exceed €400, unless otherwise agreed.
- The Visa Application Fee Grant cannot exceed €80.
- Reimbursements will be made **after** the conference via PayPal or bank
  transfer (SEPA and Swift). It’s not possible to be reimbursed before the
  conference or by other means, such as cash.
- Note that we might experience difficulty transacting with certain countries
  due to sanctions or international regulations.
- Reimbursements will only be made after we’ve received your
  travel/accommodation receipts/invoices. The deadline to submit your

>   receipts/invoices is **20 July 2025** [Anywhere on Earth][AoE].

Reviewer: 20 July feels a bit tight, thinking that people attending the sprints might be traveling on this day. I think is better if you give them 2-3 days more.

Author: agreed, I will check with the team on this and update accordingly.

- Please make sure that your receipts/invoices contain the information necessary
  for us to validate them. At the very least, they should contain your name, the
  date, the amount, and specify what the receipt/invoice is for.
- Reimbursements will only be made to individuals who actually attended the
  conference. Please include a **photo of your conference badge** with your
  receipts/invoices.
- Please consider the risk of travel restrictions: choose flexible booking
  options that allow you to cancel in case you can’t actually make it to the
  conference.
- If you’ve applied for a grant or accepted a grant, but cannot make it to the
  conference, please contact us at [finaid@europython.eu][email-finaid] as soon as possible. This
  allows us to reallocate your grant to someone else.

---

## Additional question

If you have any additional questions for the Financial Aid team, don't hesitate
to contact us via [finaid@europython.eu][email-finaid].

See you at EuroPython in Prague!

---

> [application-form]: https://forms.gle/xJ564JFBKLsGrtqz9

Reviewer 3: Remember to publish the form when you launch :)

Author: Thank you so much for the feedback!

Author: 1. Added new section Financial Aid in `Info & Support` like last year website
2. Updated 7 march as Financial Aid Programme start date
3, 4 I'll update in sometime after having discussion with the team
thank you!

Reviewer 3: - Updated 2 in https://github.com/EuroPython/website/pull/1037/commits/f63f2e44717370bf9c66707041689406fbc80b1b
- Updated 3 in https://github.com/EuroPython/website/pull/1037/commits/e9043fb2b74ee85ffde49b3d87d22e221a8c3679
- Updated 4 in https://github.com/EuroPython/website/pull/1037/commits/96aa8abbecfba1169d712952928e082d0e3f1bd0

[privacy-policy]: https://www.europython-society.org/privacy/
[AoE]: https://en.wikipedia.org/wiki/Anywhere_on_Earth
[email-finaid]: mailto:finaid@europython.eu

> [email-sponsoring]: mailto:sponsoring@europython.eu

Reviewer 3: ```suggestion
[email-sponsoring]: mailto:sponsors@europython.eu
```
